### PR TITLE
Sass too

### PR DESCRIPTION
**Pull request by ricardoom**
@aslagle I set up sass, check it out.

### CODE REVIEW

File: examples/alice/css/core.css [hunk 1]
```diff
@@ -1,75 +1,69 @@
-@charset "UTF-8";
```
Comment (aslagle) [inline examples/alice/css/core.css:1]: Just fyi, the files in examples/alice/css are extracted from an epub file, and we won't have control over them when we use this with other books. I'd probably leave them alone but it doesn't really matter.

Comment (ricardoom) [reply examples/alice/css/core.css:1]: so any global stylings need to happen in the root then, yeah?

Comment (aslagle) [reply examples/alice/css/core.css:1]: Any styling changes _within the book content_ have to be done in js, the way I have it set up now. For the nav and any of the book controls, I think we should include a new css file from index.html.

**Comment by ricardoom** [comment]
@aslagle could you take a look at this? Do you think it can be merged? I could branch off of this and work on the node-sass bit. not sure about the sequencing...

**Comment by aslagle** [comment]
I would like to get the node-sass commands in first, if you don't mind, to make sure I can build it.

I also don't think I want the NYPL reds in here since this is a general library. If we do an NYPL-branded version I'd make another repo that uses this library but has its own colors. The NYPL Monochrome is good though.

File: src/styles/css/controls.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  width: 12.5rem;
+  height: 12.5rem;
+  text-align: center;
+  left: calc(50% - 6.25rem);
+  top: calc(50% - 6.25rem);
+  background-color: #76777b; }
+
+.controls-view {
+  position: fixed;
+  top: 0.125rem;
```
Comment (aslagle) [inline src/styles/css/controls.css]: This is making the settings view cover the nav, so you can't click it again to close the settings, which right now is the only way to close the settings.

Comment (ricardoom) [reply src/styles/css/controls.css]: ok. i fixed the issue where the settings dialog was hiding the settings link.

**Pull request closed without merging**